services/merak-topo/main.go

```diff
@@ -39,6 +39,4 @@ func main() {
 		log.Fatalf("fail to serve: %v", err)
 	}
 
-	// log.Printf("DB connected")
-
```

Comment:
@zzengcs remove it if you don't need it.

Reply:
It has been removed